.github/workflows/windows-ci.yml

```diff
@@ -53,7 +53,7 @@ jobs:
     with:
       # It would be desirable to have a matrix of ubuntu OS for this job, but due to the issue opened in this ticket:
       # https://github.com/orgs/community/discussions/128118 , it has been set as a single OS job.
-      os-version: ${{ inputs.os-version || 'windows-2019' }}
+      os-version: ${{ inputs.os-version || 'windows-2022' }}
       vs-toolset: ${{ inputs.vs-toolset || 'v142' }}
```

Comment:
We should also add a matrix with `v143` in the PRs workflow if we are introducing `v143` in the nighties